Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -3166,9 +3162,9 @@ view: |-
 inputs:
 - key: OwnerNotificationSubject
   value:
-    simple: "A new security risk was identified on an external service owned by your team"
+    simple: A new security risk was identified on an external service owned by your team
```

Comment:
```suggestion
    simple: A new security risk was identified on an external service owned by your team.
```

Reply:
This is an email/ticketing system subject so I don't think it will need a period